package.json

```diff
@@ -51,6 +51,11 @@
                         "Swipes navigate between recent code and files - like Xcode.",
                         "Swipes navigate between recent files."
                     ]
+                },
+                "swipeToNavigate.directionReverse": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Controls whether swipe right goes forward or backward. Default is going forward."
```

Comment:
I find this ambiguous. Unfortunately I don't have a better suggestion. 
Maybe an improvement would be to declare what is default and what this toggles. 
```
                    "description": "Enable to make swiping from left to right as going to previous. Default is swiping right to left."
```

I am open for suggestions.